src/main/java/com/cloudogu/smeagol/wiki/infrastructure/WikiResource.java

```diff
@@ -0,0 +1,22 @@
+package com.cloudogu.smeagol.wiki.infrastructure;
+
+import org.springframework.hateoas.ResourceSupport;
+
+public class WikiResource extends ResourceSupport {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS2160)